routes/resourcePages.js

```diff
@@ -68,7 +69,9 @@ async function readResourcePage (req, res, next) {
   const IsomerFile = new File(accessToken, siteName)
   const resourcePageType = new ResourcePageType(resourceRoomName, resourceName)
   IsomerFile.setFileType(resourcePageType)
-  const { sha, content } = await IsomerFile.read(pageName)
+  const { sha, content: encodedContent } = await IsomerFile.read(pageName)
+
+  const content = Base64.decode(encodedContent)
```

Comment:
should this be `base64` not `Base64`

Reply:
Resolved in https://github.com/isomerpages/isomercms-backend/pull/138/commits/553ea4055480ba67a59486bde57d4e9b508a82c1